### DISCUSSION TITLE
OpenMage and Magento 1.x EE

### DISCUSSION THREAD

**Discussion by exhauster** [OpenMage/magento-lts#3873] [Questions & Answers] [answered]
Hello! I would have a short question - from the information I have seen on the website, OpenMage is based on Magento CE. There are several tutorials migrating from Magento to OpenMage. I did not find any information right now if OpenMage can also be used to migrate from Magento 1.x EE. Is that somehow possible?

**Comment by Flyingmana** [comment, marked as answer]
it is possible, but more complicated as EE also contains database modifications and several extra features.

If you search for migrating magento EE towards magento CE you will likely find more informations.


There is also the theoretical option of using OpenMage with the magento EE modules, but I think we did break to much for it to still work this way.

**Reply by exhauster** [reply]
Thank you very much for quick answering my question!

**Reply by luigifab** [reply]
Are you using EE? Did you tried the upgrade/migration?
